src/ca/mcgill/cs/jetuml/views/StringViewer.java

```diff
@@ -42,33 +52,61 @@
 	private static final FontMetrics FONT_BOLD_METRICS = new FontMetrics(FONT_BOLD);
 	
 	private static final Dimension EMPTY = new Dimension(0, 0);
-	private static final int HORIZONTAL_TEXT_PADDING = 7;
-	private static final int VERTICAL_TEXT_PADDING = 7;
+	private static final int DEFAULT_HORIZONTAL_TEXT_PADDING = 7;
+	private static final int DEFAULT_VERTICAL_TEXT_PADDING = 7;
+	
+	private static final Map<Set<Object>, StringViewer> STORE = new HashMap<Set<Object>, StringViewer>();
 	

```

Comment:
It's a bit counter-intuitive to have a set of objects of different types, and leads to loose typing. I would recommend instead a two-level map, the top level being alignment.

Reply:
I changed this in my most recent commit